### DISCUSSION TITLE
Swap Body weight with Nutrition in header.

### DISCUSSION THREAD

**Discussion by ImTheTom** [wger-project/wger#1027] [General]
Hey,

How would you feel about swapping the body weight drop down in the header with the nutrition drop down in the web app.

This would mean the header has the same ordering as the cards in the dashboard on the home page and the same order as the tabs in app.

Cons would include users who are already used to the ordering.

**Comment by rolandgeider** [comment]
yeah, you are right

**Reply by rolandgeider** [reply]
Made a new issue for this so it doesn't get lost

https://github.com/wger-project/wger/issues/1028
